### PR TITLE
Meshfetchorders

### PR DESCRIPTION
**Pull request by joeykrug**

### CODE REVIEW

**Review by ghost — COMMENTED** [review]
This pull request has been automatically deployed to [FeaturePeek](https://dashboard.featurepeek.com). 🔮
Your deployment will be kept up-to-date with this pull request's latest changes.

**https://peek.run/2n5h6lz**

-------

*Please read [our docs](https://docs.featurepeek.com/configuring-peek-yml/) for more configuration details.*

File: packages/augur-sdk/src/api/ZeroX.ts [hunk 1]
```diff
@@ -179,14 +180,34 @@ export class ZeroX {
   }
 
   async getOrders(): Promise<OrderInfo[]> {
-    // TODO when browser mesh supports this back out to using it if _rpc not provided
-    if (!this.rpc) {
-      throw Error('getOrders is not supported on browser mesh');
+    var response;
+    if (this.rpc) {
```
Comment (pgebheim) [inline packages/augur-sdk/src/api/ZeroX.ts:184]: Is the intention to have browser mesh just be a fallback for this?

File: packages/augur-sdk/src/api/ZeroX.ts [hunk 2]
```diff
@@ -179,14 +180,34 @@ export class ZeroX {
...
+      response = await this.rpc.getOrdersAsync();
+    }
+    else {
+      response = await this.getMeshOrders();
```
Comment (pgebheim) [inline packages/augur-sdk/src/api/ZeroX.ts:188]: this.rpc being unset does *not* mean this.mesh is set

File: packages/augur-sdk/src/api/ZeroX.ts [hunk 3]
```diff
@@ -179,14 +180,34 @@ export class ZeroX {
...
 
   async getOrders(): Promise<OrderInfo[]> {
-    // TODO when browser mesh supports this back out to using it if _rpc not provided
-    if (!this.rpc) {
-      throw Error('getOrders is not supported on browser mesh');
+    var response;
+    if (this.rpc) {
+      response = await this.rpc.getOrdersAsync();
+    }
+    else {
```
Comment (pgebheim) [inline packages/augur-sdk/src/api/ZeroX.ts]:
```suggestion
    else {
```
```suggestion
    else if (this.mesh) {
```

**Review by pgebheim — APPROVED** [review]

File: packages/augur-sdk/src/api/ZeroX.ts (hunk 1 above)
Comment (joeykrug) [reply packages/augur-sdk/src/api/ZeroX.ts:184]: yeah

**Pull request merged**